scripts/static_content_generation/Generate_StaticContent_MD.pl

```diff
@@ -0,0 +1,389 @@
+		}
+	}
+
+
+	## Test if the JSON file is empty, if yes raise a warning and produce a placeholder summary in about.md static file to be filled in manually later. 
+	unless (-z $json_file_path){
+		$has_wiki_json_data = 1;
+	}
+	else{
+		print BOLD RED "Wikipedia summary information NOT FOUND in JSON file: $json_file_path\t<----Skipping parse. Manual editing required for this species !!\n";
+
+		$about_content = "**About *$scientific_name***
+-------------------------
+!!!!!! PLACEHOLDER SUMMARY. MANUAL EDIT REQUIRED HERE !!!!!!
+
+Picture credit: [LICENSE TYPE]() via Wikimedia Commons [(Image source)]()
+
+Taxonomy ID [$taxa_id](https://www.uniprot.org/taxonomy/${taxa_id})
+
+(Text from [Wikipedia](https://en.wikipedia.org/).)
+
+**More information**
+General information about this species can be found in [Wikipedia](https://en.wikipedia.org/wiki/${species_name})\n";
+	
+	print OUT_ABOUT $about_content;
+	}
+
+#check if a file has JSON before attempting to parse it
+if ($has_wiki_json_data == 1){
+		my $fixed_json;
+		$fixed_json = "${json_file_path}.fixed";
+		my $file_contents = `cat $json_file_path`;
+		chomp $file_contents;
+		$fixed_json =~ s/\.json\.fixed/.fixed.json/;
+			open(TEMPOUT, ">$fixed_json") || die "Can't open $fixed_json\n";
+			print TEMPOUT "[".$file_contents."]";
+			close TEMPOUT;
+	
+		my $json_text = do {
+		open(my $json_fh, "<:encoding(UTF-8)", $fixed_json) or die ("Can't open JSON file: $fixed_json: $!\n");
+		local $/;
+		<$json_fh>
+		};
+	
+		# Create JSON object and data vars we want to capture
+		my $json_ob = JSON->new;
+		my $jsondata = $json_ob->decode($json_text);
+		# print Dumper(\$jsondata);
+	
+		# Extract data from json.
+		foreach (@$jsondata){
+			$extract = $_->{extract};
+			$extract =~ s/$scientific_name/\*$scientific_name\*/;
+			$extract =~ s/$common_name/$common_name (*$scientific_name*)/;
+			$jpg_source = $_->{'thumbnail'}{'source'};
+			$wiki_url = $_->{'content_urls'}{'desktop'}{'page'};
+		}
+		#Wrap the summary text
+		($wiki_about = $extract) =~ s/(.{0,$wrap_at}(?:\s|$))/$1\n/g;
+
+		# If jpg source is not defined
+		unless ($jpg_source){
+			$jpg_source="";
+		}
+
+$about_content = "**About *$scientific_name***
+-------------------------
+$wiki_about
+Picture credit: [LICENSE TYPE]() via Wikimedia Commons [(Image source)]($jpg_source)
+
+Taxonomy ID [$taxa_id](https://www.uniprot.org/taxonomy/${taxa_id})
+
+(Text from [Wikipedia](https://en.wikipedia.org/).)
+
+**More information**
+General information about this species can be found in [Wikipedia]($wiki_url)\n";
+	
+print OUT_ABOUT $about_content;
+
+	}
+	if ( $annotation_source eq "DToL" ){
```

Comment:
@leannehaggerty Can you have a look here

Reply:
Seems sensible to me. 
There's a better description of the annotation methods for non-verts here https://rapid.ensembl.org/info/genome/genebuild/anno.html, if you'd rather use that, that paper is really outdated! But if you're trying to be consistent, it's not important, what you've said is still valid.

Reply:
I think given we are only dealing with non-verts this suggestion is worth implementing. I will make the updates to incorporate this link. Though I assume this page will go-away at some point when rapid winds down